components/Article/Codebox/index.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import Codebox from './index';
+
+export default { component: Codebox };
+
+const code = 'const a = 1;';
+
+export const Default = {
+  args: {
+    children: <code className="language-js">{code}</code>,
```

Comment:
```suggestion
    children: <pre className="language-js">{code}</pre>,
```

Reply:
If it is `code` then it should be the InLineCode. Also could you add another story here for the InLine code?